components/layout_thread/lib.rs

```diff
@@ -244,7 +244,6 @@ impl LayoutThreadFactory for LayoutThread {
 
     /// Spawns a new layout thread.
     fn create(id: PipelineId,
-              top_level_browsing_context_id: Option<BrowsingContextId>,
```

Comment:
I remembered why we wanted this, which is for crash reporting, so that if a layout panic happens we know which top-level browsing context to blame.